[2021-10-21 17:37:20] [ERROR]: Protocol -> unknown group message type: 12 
[2021-10-21 17:37:20] [ERROR]: Protocol -> unknown group message type: 12 
[2021-10-21 17:37:20] [ERROR]: Protocol -> unknown group message type: 12 
[2021-10-21 17:37:22] [WARNING]: lambda 初始化失败: Post "http:///runtime/init/ready": http: no Host in request URL 
[2021-10-21 17:37:22] [WARNING]: 警告: pprof 服务不支持鉴权, 请不要运行在公网. 
[2021-10-21 17:37:24] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-21 17:37:29] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-21 17:40:19] [WARNING]: 监听反向WS API时出现错误: read tcp 127.0.0.1:61666->127.0.0.1:8080: wsarecv: An existing connection was forcibly closed by the remote host. 
[2021-10-21 17:40:22] [WARNING]: 向WS服务器 127.0.0.1:8080 推送Event时出现错误: write tcp 127.0.0.1:61666->127.0.0.1:8080: use of closed network connection 
[2021-10-21 17:42:12] [WARNING]: 获取信息时出现错误: leveldb: not found id: -1746266094 
[2021-10-21 17:43:28] [WARNING]: 监听反向WS API时出现错误: read tcp 127.0.0.1:53163->127.0.0.1:8080: wsarecv: An existing connection was forcibly closed by the remote host. 
[2021-10-21 17:43:32] [WARNING]: 向WS服务器 127.0.0.1:8080 推送Event时出现错误: write tcp 127.0.0.1:53163->127.0.0.1:8080: use of closed network connection 
[2021-10-21 17:44:11] [WARNING]: 监听反向WS API时出现错误: read tcp 127.0.0.1:53194->127.0.0.1:8080: wsarecv: An existing connection was forcibly closed by the remote host. 
[2021-10-21 17:44:12] [WARNING]: 向WS服务器 127.0.0.1:8080 推送Event时出现错误: write tcp 127.0.0.1:53194->127.0.0.1:8080: wsasend: An existing connection was forcibly closed by the remote host. 
[2021-10-21 17:46:24] [WARNING]: 监听反向WS API时出现错误: read tcp 127.0.0.1:53198->127.0.0.1:8080: wsarecv: An existing connection was forcibly closed by the remote host. 
[2021-10-21 17:46:27] [WARNING]: 向WS服务器 127.0.0.1:8080 推送Event时出现错误: write tcp 127.0.0.1:53198->127.0.0.1:8080: use of closed network connection 
[2021-10-21 17:47:44] [WARNING]: 监听反向WS API时出现错误: read tcp 127.0.0.1:52906->127.0.0.1:8080: wsarecv: An existing connection was forcibly closed by the remote host. 
[2021-10-21 17:47:47] [WARNING]: 向WS服务器 127.0.0.1:8080 推送Event时出现错误: write tcp 127.0.0.1:52906->127.0.0.1:8080: use of closed network connection 
[2021-10-21 17:50:09] [WARNING]: 监听反向WS API时出现错误: websocket: close 1006 (abnormal closure): unexpected EOF 
[2021-10-21 17:50:12] [WARNING]: 向WS服务器 127.0.0.1:8080 推送Event时出现错误: write tcp 127.0.0.1:65532->127.0.0.1:8080: use of closed network connection 
[2021-10-21 17:50:17] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-21 17:50:22] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-21 17:50:28] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-21 17:50:33] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-21 17:50:38] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-21 17:50:43] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-21 17:50:48] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-21 17:50:53] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-21 17:50:58] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-21 17:51:03] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-21 17:51:08] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
